src/main/webapp/app/exam/manage/exams/exam-exercise-import/exam-exercise-import.component.html

```diff
@@ -0,0 +1,140 @@
+<div class="alert alert-info" [hidden]="containsProgrammingExercises.size === 0">
```

Comment:
I suggest to use ngIf instead of hidden: https://stackoverflow.com/questions/43034758/what-is-the-difference-between-ngif-and-hidden